shared/features/feedback-drawer/feedback-drawer.tsx

```diff
@@ -14,8 +15,11 @@ export function FeedbackDrawer({ state }: { state: ReturnType<typeof useFeedback
     <Drawer
       isOpen={isOpen}
       onOpenChange={setIsOpen}
-      hideHeader
+      header={{
+        startContent: <Typo size={"2xl"} variant="brand" translate={{ token: "feedbackDrawer:title" }} />,
+      }}
       classNames={{
+        base: "bg-[#181818]",
```

Comment:
There is no variable ?

Reply:
It's to match the color set in the Fillout admin that matches the old app